corehq/apps/ota/views.py

```diff
@@ -58,9 +59,11 @@ def get_restore_response(domain, couch_user, since=None, version='1.0',
                             status=401)
 
     project = Domain.get_by_name(domain)
+    app = Application.objects.get(app_id) if app_id else None
```

Comment:
have you run/tested this? `Application` is a couch model so I don't think the `.objects` call is right/will work.